assets/airports/lkpr.json

```diff
@@ -184,319 +233,2405 @@
+            "rwy": {
+                "LKPR12": []
+            },
+            "draw": [["LOMKI", "PR707", "PR956", "PR954", "SOMIS"]]
+        },
+        "LOMKI6S": {
+            "icao": "LOMKI6S",
+            "name": "Lomki Six Sierra",
+            "entryPoints": {
+                "LOMKI": ["LOMKI"]
+            },
+            "body": [["PR511", "S250-"], "PR512", ["PR513", "S220-"], "ERASU", "#061"],
+            "rwy": {
+                "LKPR24": []
+            },
+            "draw": [["LOMKI", "PR511", "PR512", "PR513", "ERASU"]]
+        },
+        "LOMKI6T": {
+            "icao": "LOMKI6T",
+            "name": "Lomki Six Tango",
+            "entryPoints": {
+                "LOMKI": [["LOMKI", "S250-"]]
+            },
+            "body": [["PR707", "S220-"], "@BAROX"],
+            "rwy": {
+                "LKPR06": []
+            },
+            "draw": [["LOMKI", "PR707", "BAROX"]]
+        },
+        "LOMKI7R": {
+            "icao": "LOMKI7R",
+            "name": "Lomki Seven Romeo",
+            "entryPoints": {
+                "LOMKI": ["LOMKI"]
+            },
+            "body": [["PR511", "S250-"], "PR512", ["PR914", "S220-"], "PR915", "ARVEG", "#123"],
+            "rwy": {
+                "LKPR06": []
+            },
+            "draw": [["LOMKI", "PR511", "PR512", "PR914", "PR915", "ARVEG"]]
+        },
+        "VLM2S": {
+            "icao": "VLM2S",
+            "name": "Vlasim Two Sierra",
+            "entryPoints": {
+                "VLM": ["VLM"]
+            },
+            "body": [["PR522", "S250-"], ["PR523", "S220-"], "RATEV", "#061"],
+            "rwy": {
+                "LKPR24": []
+            },
+            "draw": [["VLM", "PR522", "PR523", "RATEV"]]
+        },
+        "VLM3T": {
+            "icao": "VLM3T",
+            "name": "Vlasim Three Tango",
+            "entryPoints": {
+                "VLM": ["VLM"]
+            },
+            "body": ["PR721", ["PR722", "S250-"], "PR723", "PR523", "PR521", "AKEVA", "#241"],
+            "rwy": {
+                "LKPR06": []
+            },
+            "draw": [["VLM", "PR721", "PR722", "PR723", "PR523", "PR521", "AKEVA"]]
+        },
+        "VLM3P": {
+            "icao": "VLM3P",
+            "name": "Vlasim Three Papa",
+            "entryPoints": {
+                "VLM":  ["VLM"]
+            },
+            "body": [["PR950", "S250-"], "PR951", ["PR952", "S220-"], "PR958", "EVEMI", "#303"],
+            "rwy": {
+                "LKPR12": []
+            },
+            "draw": [["VLM", "PR950", "PR951", "PR952", "PR958", "EVEMI"]]
+        },
+        "VLM3R": {
+            "icao": "VLM3R",
+            "name": "Vlasim Three Romeo",
+            "entryPoints": {
+                "VLM": ["VLM"]
+            },
+            "body": ["PR904", ["@KENOK", "S220-"]],
+            "rwy": {
+                "LKPR30": []
+            },
+            "draw": [["VLM", "PR904", "KENOK"]]
         }
     },
     "spawnPatterns": [
```

Comment:
If possible, can these airlines within the `airlines` sections be put in lowercase? It's only for consistency, but it's fine if you don't.